File: 1844. Replace All Digits with Characters/solution.py
class Solution(object):
  def replaceDigits(self, s):
    """
    :type s: str
    :rtype: str
    """
    result = ""
    for index, value in enumerate(s):
      if index % 2 == 0:
        result += value
      else:
        result += chr(ord(s[index - 1]) + int(value))
        
    return result

File: 1844. Replace All Digits with Characters/solution.cpp
class Solution {
public:
  string replaceDigits(string s) {
    string result = "";
    for (int index = 0; index < s.size(); index++){
      int value = s[index];
      if (index % 2 == 0){
        result += value;
      }
      else{
        result += char(int(s[index - 1]) + s[index] - '0');
      }
    }

    return result;
  }
};